cmd/utils.go

```diff
@@ -318,3 +325,89 @@ func isURLContains(srcURL, tgtURL, sep string) bool {
 	}
 	return false
 }
+
+// Return the file system attribute as string; containing mode,
+// uid, gid, uname, Gname, atime, mtime, ctime and md5
+func getFileSystemAttrs(file string) (string, error) {
+	var uname, gname, atime, mtime, ctime, mode, uid, gid string
+	fi, err := os.Stat(file)
+	if err != nil {
+		return "", err
+	}
+	st := fi.Sys().(*syscall.Stat_t)
```

Comment:
Also, you need to make sure to write platform-specific code into a separate file like `stat_windows.go`, make sure that you all this code works properly on all other platforms such as FreeBSD etc as well. The best way to validate is first to check if the cross-compile works fine.